src/bindings/python/src/openvino/build_utils.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+from pathlib import Path
+
+
+def _add_openvino_libs_to_search_path() -> None:
+    """Add OpenVINO libraries to the DLL search path on Windows."""
+    if sys.platform == "win32":
+        # Installer, yum, pip installs openvino dlls to the different directories
+        # and those paths need to be visible to the openvino modules
+        #
+        # If you're using a custom installation of openvino,
+        # add the location of openvino dlls to your system PATH.
+        openvino_libs = []
+        if os.path.isdir(os.path.join(os.path.dirname(__file__), "libs")):
+            # looking for the libs in the pip installation path.
+            openvino_libs.append(os.path.join(os.path.dirname(__file__), "libs"))
+        elif os.path.isdir(os.path.join(os.path.dirname(__file__), "..", "..", "..", "Library", "bin")):
```

Comment:
don't we need to change the relative paths too?

Reply:
The function is at the same level